README.md

```diff
@@ -1,152 +1,150 @@
-# Aliyun OSS SDK for PHP
+﻿# Alibaba Cloud OSS SDK for PHP
 
 [![Latest Stable Version](https://poser.pugx.org/aliyuncs/oss-sdk-php/v/stable)](https://packagist.org/packages/aliyuncs/oss-sdk-php)
 [![Build Status](https://travis-ci.org/aliyun/aliyun-oss-php-sdk.svg?branch=master)](https://travis-ci.org/aliyun/aliyun-oss-php-sdk)
 [![Coverage Status](https://coveralls.io/repos/github/aliyun/aliyun-oss-php-sdk/badge.svg?branch=master)](https://coveralls.io/github/aliyun/aliyun-oss-php-sdk?branch=master)
 
-## 概述
+## [README of Chinese](https://github.com/aliyun/aliyun-oss-php-sdk/blob/master/README-CN.md)
 
-阿里云对象存储（Object Storage Service，简称OSS），是阿里云对外提供的海量、安全、低成本、高可靠的云存储服务。用户可以通过调用API，在任何应用、任何时间、任何地点上传和下载数据，也可以通过用户Web控制台对数据进行简单的管理。OSS适合存放任意文件类型，适合各种网站、开发企业及开发者使用。
+## Overview
 
+Alibaba Cloud Object Storage Service (OSS) is a cloud storage service provided by Alibaba Cloud, featuring a massive capacity, security, a low cost, and high reliability. You can upload and download data on any application anytime and anywhere by calling APIs, and perform simple management of data through the web console. The OSS can store any type of files and therefore applies to various websites, development enterprises and developers.
 
-## 运行环境
-- PHP 5.3+
-- cURL extension
 
-提示：
+## Run environment
+- PHP 5.3+.
+- cURL extension.
 
-- Ubuntu下可以使用apt-get包管理器安装php的cURL扩展 `sudo apt-get install php5-curl`
+Tips:
 
-## 安装方法
+- In Ubuntu, you can use the ***apt-get*** package manager to install the *PHP cURL extension*: `sudo apt-get install php5-curl`.
 
-1. 如果您通过composer管理您的项目依赖，可以在你的项目根目录运行：
+## Install OSS PHP SDK
 
-        $ composer require aliyuncs/oss-sdk-php
+- If you use the ***composer*** to manage project dependencies, run the following command in your project's root directory:
 
-   或者在你的`composer.json`中声明对Aliyun OSS SDK for PHP的依赖：
+        composer require aliyuncs/oss-sdk-php
+
+   You can also declare the dependency on Alibaba Cloud OSS SDK for PHP in the `composer.json` file.
 
         "require": {
             "aliyuncs/oss-sdk-php": "~2.0"
         }
 
-   然后通过`composer install`安装依赖。composer安装完成后，在您的PHP代码中引入依赖即可：
+   Then run `composer install` to install the dependency. After the Composer Dependency Manager is installed, import the dependency in your PHP code: 
```

Comment:
After the Composer Dependency Manager is installed
->
After the Composer Dependency Manager installed